File: car-lease-service/src/main/java/org/jesperancinha/car/lease/dto/CarDto.java
package org.jesperancinha.car.lease.dto;

import lombok.Data;

@Data
public class CarDto {
    private Long id;
    private String make;
    private String model;
    private String version;
    private Long numberDoors;
    private Long co2Emission;
    private Long grossPrice;
    private Long netPrice;
}

File: car-lease-service/src/main/kotlin/org/jesperancinha/car/lease/dto/CarDto.kt
package org.jesperancinha.car.lease.dto

data class CarDto(
    val id: Long? = null,
    val make: String? = null,
    val model: String? = null,
    val version: String? = null,
    val numberDoors: Long? = null,
    val co2Emission: Long? = null,
    val grossPrice: Long? = null,
    val netPrice: Long,
    val millage: Long
)